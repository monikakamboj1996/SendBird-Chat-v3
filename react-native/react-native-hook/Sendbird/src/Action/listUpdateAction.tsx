export const ListUpdate = list => {
  return {
    type: 'List_Update',
    list: list,
  };
};
