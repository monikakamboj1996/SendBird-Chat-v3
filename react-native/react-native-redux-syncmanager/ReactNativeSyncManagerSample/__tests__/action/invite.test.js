import 'react-native';
import React from 'react';
import Action from '../../src/action/invite';

describe('action/invite', () => {
  it('fetchUsers', () => {
    const result = Action.fetchUsers([{ nickname: 'test' }, { nickname: 'test 2' }]);
    expect(result).toMatchSnapshot();
  });
});
