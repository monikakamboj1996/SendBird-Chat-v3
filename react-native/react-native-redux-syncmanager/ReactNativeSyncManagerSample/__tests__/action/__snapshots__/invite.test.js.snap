// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`action/invite fetchUsers 1`] = `
Object {
  "payload": Object {
    "users": Array [
      Object {
        "nickname": "test",
      },
      Object {
        "nickname": "test 2",
      },
    ],
  },
  "type": "user-fetch",
}
`;
